src/support/generation.c

```diff
@@ -169,7 +168,7 @@ __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
 #define WT_GEN_DRAIN_TIMEOUT_MIN 4
                 if (time_diff_ms > minutes * WT_MINUTE * WT_THOUSAND) {
                     __wt_verbose_notice(session, WT_VERB_GENERATION,
-                      "%s generation drain waited %u minutes", __gen_name(which), minutes);
+                      "%s generation drain waited %" PRIu64 "minutes", __gen_name(which), minutes);
```

Comment:
this is missing a space before "minutes" now

Reply:
Fixed that!